make/clean.mk

```diff
@@ -1,4 +1,12 @@
 .PHONY: clean
 clean:
 	$(Q)-rm -rf ${V_FLAG} $(OUT_DIR) ./vendor
-	$(Q)go clean ${X_FLAG} ./...
\ No newline at end of file
+	$(Q)go clean ${X_FLAG} ./...
+
+.PHONY: clean-resources
+clean-resources:
+	$(Q)echo "cleaning resources in $(HOST_NS) and $(MEMBER_NS)..."
+	$(Q)oc get usersignups -n $(HOST_NS) -o name | xargs oc delete -n $(HOST_NS)
+	$(Q)oc get pods -n $(HOST_NS) -l name=host-operator -o name | xargs oc delete -n $(HOST_NS)
+	$(Q)oc get pods -n $(MEMBER_NS) -l name=member-operator -o name | xargs oc delete -n $(MEMBER_NS)
+	$(Q)oc get clusterresourcequotas -l "toolchain.dev.openshift.com/provider"=codeready-toolchain -o name| xargs oc delete
```

Comment:
I wrote it quickly - there is maybe a way to make it nicer :-) 
```suggestion
.PHONY: clean-toolchain-resources-in-dev
clean-toolchain-resources-in-dev:
	$(Q)echo "cleaning resources in $(DEV_HOST_NS) and $(DEV_MEMBER_NS)..."
	$(Q)oc get usersignups -n $(DEV_HOST_NS) -o name | xargs -i oc delete -n $(DEV_HOST_NS) {}
	$(Q)for CRD in `oc get crd -o name -n $(DEV_HOST_NS) | grep toolchain`; do \
		echo "deleting $${CRD} in $(DEV_HOST_NS)"; \
		CRD_NAME=`oc get $${CRD} -n $(DEV_HOST_NS) --template '{{.metadata.name}}'`; \
		oc get $${CRD_NAME} -n $(DEV_HOST_NS) -o name | xargs -i oc delete {}; \
	done
	$(Q)for CRD in `oc get crd -o name -n $(DEV_MEMBER_NS) | grep toolchain`; do \
		echo "deleting $${CRD} in $(DEV_MEMBER_NS)"; \
		CRD_NAME=`oc get $${CRD} -n $(DEV_MEMBER_NS) --template '{{.metadata.name}}'`; \
		oc get $${CRD_NAME} -n $(DEV_MEMBER_NS) -o name | xargs -i oc delete {}; \
	done
	$(Q)oc get clusterresourcequotas -l "toolchain.dev.openshift.com/provider"=codeready-toolchain -o name -n $(DEV_MEMBER_NS) | xargs -i oc delete {}
	$(Q)oc get pods -n $(DEV_HOST_NS) -l name=host-operator -o name | xargs -i oc delete -n $(DEV_HOST_NS) {}
	$(Q)oc get pods -n $(DEV_MEMBER_NS) -l name=member-operator -o name | xargs -i oc delete -n $(DEV_MEMBER_NS) {}
```

Reply:
This is a comment to the @MatousJobanek suggestion :)

- `clean-toolchain-resources-in-dev` is not an intuitive name. Let's add a detailed comment explaining what this target does. It might be worth adding it to readme too...
- You are deleting toolchain CRDs from the namespaces but CRDs are cluster scoped resources. So, that namespace iteration and `-n` params are redundant.
- Wouldn't deleting CRDs break the operators?
- What about deleting tiers?

I would move this target out of this PR to a separate one. Don't want to block this PR because of that.

Reply:
crated PR for that: https://github.com/codeready-toolchain/toolchain-e2e/pull/100